restapi/src/main/java/io/stargate/web/resources/Db.java

```diff
@@ -102,4 +102,8 @@ public DocumentDB getDocDataStoreForToken(String token, int pageSize, ByteBuffer
     return new DocumentDB(
         DataStore.create(persistence, storedCredentials.getRoleName(), parameters));
   }
+
+  public boolean isDse() {
+    return persistence.name().equals("DataStax Enterprise");
```

Comment:
using a string is kind of brittle, is there a better way?

Reply:
Not unless you want to query system.local. We could define some constants in persistence-api